packages/web3-providers/src/twitter/index.ts

```diff
@@ -67,13 +80,15 @@ async function getTokens(operationName?: string) {
     const queryId = operationName
         ? getScriptContentMatched(mainContent ?? '', new RegExp(`queryId:"([^"]+)",operationName:"${operationName}"`))
         : undefined
-
-    return {
+    tokens = {
         bearerToken,
         queryToken,
         csrfToken,
         queryId,
     }
+    tokenCache.set('tokens', tokens)
```

Comment:
Why LRU?

Reply:
Maybe an in-memory local variable is enough.

Reply:
Well, LRU can control cache expiration.

Reply:
use LRU but you only store 1 item? that's strange

Reply:
Alright, it's better to isolate it. If you mix it into another cache object, when the number of items reached `max`, our cached tokens might be cleared.